src/react/components/searchAutocomplete.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import sanitize from "sanitize-html";
+
+const searchAutocomplete = ({
+  autocompleteSuggestions,
+  showAutocomplete,
+  onSelect,
+  selectedSuggestion,
+  setSelectedSuggestion,
+  autocompleteContainerClasses,
+  autocompleteLinkClasses,
+  autocompleteLinkFocusClasses,
+}) => (
+  <div
+    className={`algolia-search--autocomplete-container ${autocompleteContainerClasses}
+    ${showAutocomplete && autocompleteSuggestions.length ? "" : "hidden"}`}
+  >
+    {Array.isArray(autocompleteSuggestions) && (
+      <ul className="list-unstyled pl-0" role="listbox">
+        {autocompleteSuggestions.map((suggestion, index) => (
+          // eslint-disable-next-line jsx-a11y/click-events-have-key-events
+          <li
+            key={`autocomplete-item-${suggestion.objectID}`}
```

Comment:
I think ObjectID can have spaces in it. Not sure if that matters here or not.

Reply:
Some updated code you could use, @nplowman 
https://github.com/SU-SWS/saa_alumni/pull/172

Reply:
White space could potentially cause issues. I can look into an alternate identifier or base64 encode the Object ID to avoid this.